source/connect/tls.txt

```diff
@@ -0,0 +1,310 @@
+.. _pymongo-tls:
+
+=============
+Configure TLS 
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: security, authentication, transport layer security, encrypt
+
+Overview
+--------
+
+In this guide, you can learn how to use the :wikipedia:`TLS <Transport_Layer_Security>`
+protocol to secure your connection to a MongoDB deployment.
+
+When you enable TLS for a connection, {+driver-short+} uses TLS to connect to the
+MongoDB deployment, verifies the deployment's certificate, and ensures that the certificate
+certifies the deployment {+driver-short+} is connecting to.
+
+To learn more about the connection options in this guide,
+see the
+`mongo_client <https://pymongo.readthedocs.io/en/latest/api/pymongo/mongo_client.html>`__
+API documentation. To learn how to configure your MongoDB deployment for TLS, see the
+:manual:`TLS configuration guide </tutorial/configure-ssl/>` in the
+MongoDB Server manual. 
+
+.. important::
+
+   A full description of TLS/SSL, PKI (Public Key Infrastructure) certificates, and
+   Certificate Authorities (CAs) is beyond the scope of this document. This page assumes prior
+   knowledge of TLS/SSL and access to valid certificates.
```

Comment:
Pulled straight from server docs

Reply:
Updated the link and moved to an API Documentation section to match the rest of our pages.

We don't maintain the pages on readthedocs, but feel free to make any updates there you think make sense!